### PR TITLE
README: Make sentences about fluency more fluent

### PR DESCRIPTION
**Pull request by kpu**
I've rewritten some sentences to be more fluent.

### CODE REVIEW

**Comment by ZJaume** [comment]
Thanks for the fluentification! I'm making a couple of comments.

File: README.md [hunk 1]
```diff
@@ -67,9 +67,9 @@ or downloaded from an external source, such as https://github.com/wooorm/diction
 You can also provide the path to the Hunspell dictionaries directories by using the dictpath atribute in `{/YOUR/INSTALLATION/PATH}/config/hunspell.yaml` (for example, `venv/lib/python3.7/site-packages/fastspell/config/hunspell.yaml` ) if you are installing from PyPI or with `setup.py`, or in `/config/hunspell.yaml` if you are running directly the code. Default path is `/usr/share/hunspell`.
 
 ## Scoring
-`monocleaner` aims at detecting poor fluent sentences in a monolingual corpus.
-It indicates the fluency of a sentence with a score between 1 and 0, where the higher score, the better the fluency is.
-Sentences considered to have obvious poor fluency are tagged with a 0.
+`monocleaner` aims aims to detect disfluent sentences in a monolingual corpus.
```
Comment (ZJaume) [inline README.md]: there's two "aims" here

File: README.md [hunk 2]
```diff
@@ -3,9 +3,9 @@
 
 ![License](https://img.shields.io/badge/License-GPLv3-blue.svg)
 
-Monocleaner is a Python tool that aims at detecting poor fluent sentences in a monolingual corpus.
-It indicates the fluency of a sentence with a score between 1 and 0, where the higher score, the better the fluency is.
-Sentences considered to have obvious poor fluency are tagged with a 0.
+Monocleaner is a Python tool that aims to detect disfluent sentences in a monolingual corpus.
+Each sentence assigned a fluency score between 0 and 1, with higher scores indicating more fluency.
```
Comment (ZJaume) [inline README.md]: Shouldn't this be something like "Each sentence is assigned a fluency..."?

**Review by ZJaume — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (kpu) [reply README.md]: You're right.  The irony branch.

File: README.md (hunk 1 above)
Comment (kpu) [reply README.md]: Fixed

**Pull request merged**